### ISSUE TITLE
Text formatting (italics/bold) within Quest editor is inconsistent

### ISSUE THREAD

**Issue by Liquidlogic1** [NPBruce/valkyrie#20]
When I am using the <b) and <i) in the text boxes of the editor, it works well when I am typing on-the-fly, but after going back to make changes to the text(after exiting, saving, or moving the cursor with arrow keys), the italics and bold formatting is not always consistent.

Is there a proper way to insert the commands for bold and italics? Do I need to type <i) Text </i) to make the entire line read italics? For example, do I need to reestablish the < i) each time I end a line with "\n"?

**Comment by NPBruce** [comment]
Whenever you have fully closed symbols it will magic into rich formatting. If you have unfinished tags it will expand out. /n won't end the section until </i.

This whole system is pretty bad and at some point needs to be redone. At this point it is low priority I recommend editing dialogue in a text editor.

**Comment by NPBruce** [comment]
I am waiting on the localisation work to review text editing.

**Comment by NPBruce** [comment]
I am going to close this as the system has been overhauled.  We can raise new issues for additional fixes/features.
